docs/source/configuration/overview.mdx

```diff
@@ -589,11 +589,10 @@ Starting with v1.49.0, the router can run a Rust-native query planner. This nati
 
 <Tip>
 
-Starting with v1.56.0, to run the most performant and resource-efficient native query planner and to disable the V8 JavaScript runtime in the router, set the following options in your `router.yaml`:
+Starting with v1.56.0, to run the most performant and resource-efficient native query planner and to disable the V8 JavaScript runtime in the router, set the following option in your `router.yaml`:
```

Comment:
`experimental_introspection_mode` was removed in router v1.57.0. 

For router v1.56, can add a note to set the additional option

```suggestion
Starting with v1.57.0, to run the most performant and resource-efficient native query planner and to disable the V8 JavaScript runtime in the router, set the following option in your `router.yaml`:
```